envoy/tests/test_bench.py

```diff
@@ -1,9 +1,8 @@
-import mock
 import pytest
 
 from datadog_checks.envoy import Envoy
 
-from .common import INSTANCES, response
```

Comment:
Small nit, but is `response` being used elsewhere? If not, we should remove it from `common.py` entirely.

Reply:
https://github.com/DataDog/integrations-core/pull/10242

Reply:
I had already removed it actually